### ISSUE TITLE
Support basic BPMN2 events

### ISSUE THREAD

**Issue by adamfowleruk** [adamfowleruk/scxmltocpf#8]
Escalation event (non timer escalation)
Error event (custom or generic)
Custom event (any event)

Treat events as being CPF states that immediately start 1 or more parallel sub processes. (multiple event listeners)

Message and Signal events both out of scope

**Comment by adamfowleruk** [comment]
Also support event based gateway in this feature.

**Comment by adamfowleruk** [comment]
Depends upon #4
